website/content/docs/k8s/helm.mdx

```diff
@@ -136,13 +137,13 @@ Use these links to navigate to a particular top-level stanza.
     and have necessary secrets, policies and roles created prior to installing Consul.
     See https://www.consul.io/docs/k8s/installation/vault for full instructions.
 
-    The Vault cluster *must* not have the Consul cluster installed by this Helm chart as its storage backend
+    The Vault cluster _must_ not have the Consul cluster installed by this Helm chart as its storage backend
     as that would cause a circular dependency.
     It can have Consul as its storage backend as long as that Consul cluster is not running on this Kubernetes cluster
```

Comment:
```suggestion
    Vault can have Consul as its storage backend, as long as that Consul cluster is not running on this Kubernetes cluster
```

Reply:
Resolved in https://github.com/hashicorp/consul-k8s/pull/1010